shop/shop-domain/build.gradle

```
@@ -0,0 +1,3 @@
+dependencies {
+    testImplementation 'org.springframework.boot:spring-boot-starter-test'
```

```suggestion
    testImplementation 'org.assertj:assertj-core:3.24.2'
```

저희 도메인 테스트에서 `AssertJ`만 추가해서 사용하기로 했던 것 같아요!